eldev-util.el

```diff
@@ -1443,5 +1520,9 @@ absolute paths, but are traced without them.  See
           (eldev--do-path-matches actual-path pattern-path-rest)
           (and actual-path-rest (eldev--do-path-matches actual-path-rest pattern-path))))))
 
+(defmacro eldev--shell-script-name ()
```

Comment:
Just use a `defun` here. Don't want difficult-to-understand errors if a Windows-compiled `.elc` file is used on Linux or vice versa.

Reply:
Done.